src/coreclr/vm/jithelpers.cpp

```diff
@@ -762,6 +748,64 @@ HCIMPL2_VV(double, JIT_DblRem, double dividend, double divisor)
 }
 HCIMPLEND
 
+/*********************************************************************/
+HCIMPL1_V(UINT64, JIT_Dbl2ULngOvf, double val)
```

Comment:
Nit: This should be together with the order Dbl -> integer helpers. Moving these two after JIT_Dbl/FltRem does not make logical sense.

Reply:
Added those changes

Reply:
I still see `JIT_Dbl2ULngOvf` after `JIT_DblRem`. It should be moved before `JIT_FltRem`

Reply:
Moved it to appropriate place with other overflow conversion helpers.